app/src/main/java/com/thinqtv/thinqtv_android/AboutUS.java

```diff
@@ -0,0 +1,88 @@
+//For the "Get Involved" -> "Organizations"
+
+package com.thinqtv.thinqtv_android;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Spinner;
+import android.widget.Toast;
```

Comment:
This and several other classes you added have unused imports that should be cleaned up.

Reply:
Removed unused classes. Thank you for reviewing!